docs/conf.py.jinja

```diff
@@ -129,7 +129,14 @@ html_theme_options = {
         "image_light": "_static/logo.svg",
         "image_dark": "_static/logo-dark.svg",
     },
-    "external_links": [],
+    "external_links": [
+        {"name": "scipp", "url": "https://scipp.github.io"},
+        {"name": "plopp", "url": "https://scipp.github.io/plopp"},
+        {"name": "scippnexus", "url": "https://scipp.github.io/scippnexus"},
+        {"name": "scippneutron", "url": "https://scipp.github.io/scippneutron"},
+        {"name": "ess", "url": "https://scipp.github.io/ess"},
+        {"name": "sciline", "url": "https://scipp.github.io/sciline"},
+    ],
```

Comment:
Can you add them to the intersphinx mapping as well? Or would it be a problem if the current project is in the list?

Reply:
I am guessing `sciline` will be heavily related to `ess*` projects? If we keep `ess` as some sort of collection of all the technique/instrument repos, then linking the two would make sense?

Reply:
I don't mind linking from `ess*` to Sciline, but the other way round does not make sense, imho, at least on the front page/main menu. If this is buried in a "Who is using Sciline" page then that is fine.

You don't see NumPy, Xarray, or Pandas linking to projects of a "random" specific area of research in a prominent place.

Reply:
Then why should Scipp link to `ess` for example?
We've had this discussion before, where you did not want to have those links in such a prominent place.
My argument back then (and still today) was that if you talk to the IDS or other scipp users, they find having those links to navigate between our projects extremely useful to have.

I'm not saying this has to be the way we do it, but people using scipp need an easy way to jump between the docs of scipp, plopp, scippneutron, ess*, scippnexus.

Reply:
> Then why should Scipp link to ess for example?

I don't think it should. What we have now is just for historic reasons.

> I'm not saying this has to be the way we do it, but people using scipp need an easy way to jump between the docs of scipp, plopp, scippneutron, ess*, scippnexus.

My fear is that by having ESS-specific (or even neutron-specific, maybe) links we will scare of potential other users. I think it gives a wrong signal ("this package is really just something ESS uses"). Or at least I have worked under the assumption that this is not the signal we want to give?